### DISCUSSION TITLE
What is the purpose of the archive section?

### DISCUSSION THREAD

**Discussion by sympmarc** [SharePoint/sp-dev-docs#8672] [General]
What is the value/purpose of the Archive section in the docs here? By this, I mean the articles starting with this one: [Overview](https://learn.microsoft.com/en-us/sharepoint/dev/general-development/sharepoint-development-overview).

Today I was trying out the Geolocation capabilities (which are woeful) for a client who asked me to look into them, and I was using the [Location and map functionality](https://learn.microsoft.com/en-us/sharepoint/dev/general-development/integrating-location-and-map-functionality-in-sharepoint) section. I found some mistakes in several of the articles and did PRs to fix them.

But are these articles even valid anymore for SharePoint Online? What is their purpose? Should they have some sort of disclaimer in them about their value? It seems that the articles I was looking at have been updated in within the last year, but do they still apply? Inquiring minds!

Tagging @andrewconnell and @VesaJuvonen - just because.

**Comment by andrewconnell** [comment]
I'll let @VesaJuvonen chime in with an official MSFT response, but my understanding & $0.02.

Old content that hasn't been touched in a long time and might not reflect the current state of the product / what's supported / what's recommended. A lot came from the migration from the previous dev docs. You might see some timestamps that show the content has been updated recently, but that's likely part of a mass update to conform to MSDOCS build rules.

IMHO, they can't be deleted as being "[not] valid anymore" as there are plenty of customers who aren't on SPO. This site isn't for SPO developer documentation, it's SharePoint developer documentation, which includes old deployable options, for instance, those like SP2013 that still can only use old WF workflow engine.

**Reply by sympmarc** [reply]
At the very least, I would think they should have a disclaimer or something which indicates which version(s) of SharePoint they apply to.

**Comment by sympmarc** [comment]
I guess there's no official response. :(
